docs/source/eFeatures.rst

```diff
@@ -82,7 +82,19 @@ The time interval between the first too peaks
     doublet_ISI = peak_time[1] - peak_time[0]
 
 
-LibV5 : all_ISI_values, inv_first_ISI, inv_second_ISI, inv_third_ISI, inv_fourth_ISI, inv_fifth_ISI, inv_last_ISI
+LibV5: all_ISI_values
+~~~~~~~~~~~~~~~~~~~~~
+
+The interspike intervals (i.e. time intervals) between adjacent peaks.
```

Comment:
This sentence reads a bit weird, maybe:
The interspike intervals, i.e., the time intervals between adjacent peaks.

Reply:
Thanks, it is now updated.